btf/ext_info.go

```diff
@@ -518,12 +520,18 @@ func (li *Line) String() string {
 
 // LineInfos contains a sorted list of line infos.
 type LineInfos struct {
```

Comment:
This may be a good opportunity to refactor to match FuncOffsets:

```go
type LineOffsets []LineOffset

type LineOffset struct {
	Offset asm.RawInstructionOffset
	Line   *Line
}
```

This avoids the need for a `Lines()` method.

Please do this in a separate commit, split off from the addition of the jited proginfo logic below.